ompi/win/win.h

```diff
@@ -152,27 +151,27 @@ OMPI_DECLSPEC int ompi_win_group(ompi_win_t *win, ompi_group_t **group);
 /* Note that the defintion of an "invalid" window is closely related
    to the defintion of an "invalid" communicator.  See a big comment
    in ompi/communicator/communicator.h about this. */
-static inline int ompi_win_invalid(ompi_win_t *win) {
-    if (NULL == win ||
-        MPI_WIN_NULL == win ||
-        (OMPI_WIN_INVALID & win->w_flags) ||
+static inline int ompi_win_invalid(ompi_win_t *win)
+{
+    if (NULL == win || MPI_WIN_NULL == win || (OMPI_WIN_INVALID & win->w_flags) ||
         (OMPI_WIN_FREED & win->w_flags)) {
         return true;
     } else {
         return false;
     }
 }
 
-static inline int ompi_win_peer_invalid(ompi_win_t *win, int peer) {
-    if (win->w_group->grp_proc_count <= peer || peer < 0) return true;
+static inline int ompi_win_peer_invalid(ompi_win_t *win, int peer)
+{
+    if (win->w_group->grp_proc_count <= peer || peer < 0)
```

Comment:
This is definitely wrong (needs curly braces if we're moving down a line)

Reply:
Ok. I was going to run clang-tidy with just the curly-brace checker after this PR. I spot-fixed this one but I will go ahead and run clang-tidy and add that commit to this PR.

Reply:
Ran clang-tidy to put braces around all single-line control statements.